content/en/tracing/connect_logs_and_traces/dotnet.md

```diff
@@ -75,61 +94,52 @@ For NLog version 4.6+:
     <attribute name="date" layout="${longdate}" />
     <attribute name="level" layout="${level:upperCase=true}"/>
     <attribute name="message" layout="${message}" />
     <attribute name="exception" layout="${exception:format=ToString}" />
   </layout>
-
-  <!-- If using version below 4.4.10, you must extract the Datadog properties individually by adding <attribute> nodes -->
-  <layout xsi:type="JsonLayout">
-    <attribute name="date" layout="${longdate}" />
-    <attribute name="level" layout="${level:upperCase=true}"/>
-    <attribute name="message" layout="${message}" />
-    <attribute name="exception" layout="${exception:format=ToString}" />
-
-    <attribute name="dd.env" layout="${mdc:item=dd.env}"/>
-    <attribute name="dd.service" layout="${mdc:item=dd.service}"/>
-    <attribute name="dd.version" layout="${mdc:item=dd.version}"/>
-    <attribute name="dd.trace_id" layout="${mdc:item=dd.trace_id}"/>
-    <attribute name="dd.span_id" layout="${mdc:item=dd.span_id}"/>
-  </layout>
 ```
+For additional examples, see the automatic trace ID injection projects [for NLog45][1] or [for NLog46][2] on GitHub.
 
-For NLog version lower than 4.0, there is no built-in JSON layout.
 
+[1]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog45Example/NLog.config
+[2]: https://github.com/DataDog/dd-trace-dotnet/blob/master/samples/AutomaticTraceIdInjection/NLog46Example/NLog.config
 {{% /tab %}}
 {{< /tabs >}}
 
 
-## Manually Inject Trace and Span IDs
+## Manually inject trace and span IDs
 
-If you prefer to manually correlate your [traces][7] with your logs and tie together data for your service,
-leverage the Datadog API to retrieve correlation identifiers:
+**Note:** If you are not using a [Datadog Log Integration][7] to parse your logs, custom log parsing rules must parse `dd.trace_id` and `dd.span_id` as strings. For information, see the [FAQ on this topic][8].
 
-- Use `CorrelationIdentifier.<FIELD>` API methods to inject identifiers at the beginning and end of each [span][8] to log (see examples below).
-- Configure MDC to use the injected keys:
+To manually correlate your [APM traces][9] with application logs:
 
-    - `dd.env` Globally configured `env` for the tracer (defaults to `""` if not set)
-    - `dd.service` Globally configured root service name (defaults to the name of the application or IIS site name if not set)
-    - `dd.version` Globally configured `version` for the service (defaults to `""` if not set)
-    - `dd.trace_id` Active Trace ID during the log statement (defaults to `0` if no trace)
-    - `dd.span_id` Active Span ID during the log statement (defaults to `0` if no trace)
+1. Reference the [`Datadog.Trace` NuGet package][10] in your project.
+
+2. Use the `CorrelationIdentifier` API to retrieve correlation identifiers and add them to the log context while a span is active.
+
+3. Configure mapped diagnostic context (MDC) to use the following injected keys:
```

Comment:
The setup with config files for log4net and NLog, and the `.Enrich.FromLogContext()` for Serilog are required for both automatic AND manual instrumentation. What do you think would be the best way to reference those from both sets of instructions?

Reply:
Got it, update incoming.